tests/test_hubspot_all_fields.py

```diff
@@ -19,6 +19,12 @@ def get_matching_actual_record_by_pk(expected_primary_key_dict, actual_records):
         can_save = True
     return ret_records
 
+FIELDS_ADDED_BY_TAP = {
+    # In 'contacts' streams 'versionTimeStamp' was not available in response to the second call.
```

Comment:
```suggestion
    # In 'contacts' streams 'versionTimeStamp' is not available in response of the second call.
```
```suggestion
    # In 'contacts' streams 'versionTimeStamp' was not available in response to the second call.
```

Reply:
Explain first call and second call in ()

Reply:
Explain first call and second call in ()

Reply:
Explained first call and second call.